R/chromote_session.R

```diff
@@ -684,6 +700,39 @@ ChromoteSession <- R6Class(
       }
     },
 
+    #' @description
+    #' Set or retrieve the `enable` command arguments for a domain. These
+    #' arguments are used for the `enable` command that is called for a domain,
+    #' e.g. `Fetch$enable()`, when accessing an event method.
+    #'
+    #' @examples
+    #' if (interactive()) {
+    #'   b <- ChromoteSession$new(
+    #'     auto_events_enable_args = list(
+    #'       Fetch = list(handleAuthRequests = TRUE)
+    #'     )
+    #'   )
+    #'
+    #'   # Get current `Fetch.enable` args
+    #'   b$auto_events_enable_args("Fetch")
+    #'   # Update the `Fetch.enable` args
+    #'   b$auto_events_enable_args("Fetch", handleAuthRequests = FALSE)
+    #' }
+    #'
+    #' @param domain A command domain, e.g. `"Fetch"`.
+    #' @param ... Arguments to use for auto-events for the domain. If not
+    #'   provided, returns the argument values currently in place for the
+    #'   domain.
+    auto_events_enable_args = function(domain, ...) {
+      dots <- dots_list(..., .named = TRUE)
+
+      if (length(dots) == 0) {
+        return(get_auto_events_enable_args(private, domain, self$parent))
+      }
+
+      set_auto_events_enable_args(self, private, domain, dots)
+    },
+
```

Comment:
Is there a way to reset the args?

Reply:
Oh, I saw later on that there is, by passing `NULL`. Can you add that to the function documentation?